assets/base/i18n/en.json

```diff
@@ -2356,26 +2356,6 @@
   "mobile.client_upgrade.no_upgrade_subtitle": "You already have the latest version.",
   "mobile.client_upgrade.no_upgrade_title": "Your App Is Up to Date",
   "mobile.client_upgrade.upgrade": "Update",
-  "mobile.combined_system_message.added_to_channel": "added to the channel",
-  "mobile.combined_system_message.added_to_team": "added to the team",
-  "mobile.combined_system_message.by_actor": " by {actor}.",
-  "mobile.combined_system_message.first_user": "{firstUser} ",
-  "mobile.combined_system_message.first_user_and": "{firstUser} and ",
-  "mobile.combined_system_message.first_user_and_second_user": "{firstUser} and {secondUser} ",
-  "mobile.combined_system_message.first_user_and_second_user_were": "{firstUser} and {secondUser} were ",
-  "mobile.combined_system_message.first_user_was": "{firstUser} was ",
-  "mobile.combined_system_message.joined_channel": "joined the channel",
-  "mobile.combined_system_message.joined_team": "joined the team",
-  "mobile.combined_system_message.left_channel": "left the channel",
-  "mobile.combined_system_message.left_team": "left the team",
-  "mobile.combined_system_message.others": "{numOthers} others ",
-  "mobile.combined_system_message.removed_from_channel": "removed from the channel",
-  "mobile.combined_system_message.removed_from_team": "removed from the team",
-  "mobile.combined_system_message.users_and_last_user": "{users} and {lastUser} ",
-  "mobile.combined_system_message.users_and_last_user_were": "{users} and {lastUser} were ",
-  "mobile.combined_system_message.were": "were ",
-  "mobile.combined_system_message.you": "You",
-  "mobile.combined_system_message.you_were": "You were ",
   "mobile.commands.error_title": "Error Executing Command",
```

Comment:
Above are no longer need.  We're using the same translations both for webapp and mobile RN.